src/utils/wrapped-big-number.js

```diff
@@ -0,0 +1,16 @@
+import logError from 'utils/log-error'
+import BigNumber from 'bignumber.js'
+
+export const WrappedBigNumber = (...args) => {
+  let newBigNumber
+  try {
+    newBigNumber = new BigNumber(...args)
+  } catch (e) {
+    logError('Error instantiating WrappedBigNumber', e)
+  }
+
+  return newBigNumber
+}
+
+// Note this is exported from here.
+export { default as BigNumber } from 'bignumber.js'
```

Comment:
Why is this re-exported?  If using BigNumber directly, it seems simpler to just `import BigNumber from 'bignumber.js'`.